google-cloud-bigtable/pom.xml

```diff
@@ -218,6 +212,16 @@
       <artifactId>grpc-testing</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.opencensus</groupId>
+      <artifactId>opencensus-impl-core</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.opencensus</groupId>
+      <artifactId>opencensus-impl</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
I have a bit of concern here. The end user will have to declare both of these dependencies. I think it might be good to directly depend only on opencensus-impl and have it pull opencensus-impl-core transitively (and exclude it from from dependency enforcer